### ISSUE TITLE
supervisors can add/edit their own mobile number

### ISSUE THREAD

**Issue by Zrrrpy** [rubyforgood/casa#1786]
Part of epic #1017, dependent on #1785

AS a **supervisor**,
I WANT to add/edit a mobile number in my profile,
SO THAT I can receive SMS notifications.

### How it works now
It doesn't! Currently, email is the only contact medium associated with user accounts.

### How it should work 
Supervisors should be able to add/edit their own mobile number under "Edit Profile".

**Comment by xihai01** [comment]
@compwron is this issue a duplicate of #1787?
